src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/UtgiftsperiodeSelect.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react';
+import {
+    EUtgiftsperiodeProperty,
+    EUtgiftsperiodetype,
+    utgiftsperiodetypeTilTekst,
+} from '../../../../App/typer/vedtak';
+import styled from 'styled-components';
+import { EnsligFamilieSelect } from '../../../../Felles/Input/EnsligFamilieSelect';
+
+const StyledSelect = styled(EnsligFamilieSelect)`
+    align-items: start;
+    min-width: 140px;
+    max-width: 200px;
+`;
+
+interface Props {
+    periodetype: EUtgiftsperiodetype | '' | undefined;
+    oppdaterUtgiftsperiodeElement: (
+        property: EUtgiftsperiodeProperty,
+        value: string | number | undefined
+    ) => void;
+    lesevisning: boolean;
+    feil?: string;
+}
+
+const valgbarePeriodetyper = [
+    EUtgiftsperiodetype.ORDINÆR,
+    EUtgiftsperiodetype.OPPHØR,
+    EUtgiftsperiodetype.SANKSJON_1_MND,
+];
+
+const UtgiftsperiodeSelect: FC<Props> = (props: Props) => {
```

Comment:
Kanskje
```suggestion
const PeriodetypeSelect: FC<Props> = (props: Props) => {
```
Jeg tolker utgiftsperiode som "hele raden"

Reply:
Fixd [48463e6](https://github.com/navikt/familie-ef-sak-frontend/pull/1942/commits/48463e6edd8338ea241ad077bf0d9a6b04fb19d0)